.github/workflows/smoketest.yml

```diff
@@ -449,3 +450,84 @@ jobs:
         sudo update-locale
         tar --zstd -xf install.tgz
         install/opt/HPCCSystems/bin/unittests
+
+  # NB: this doesn't really need the post-setup data files included in install.tgz
+  # but as this is relatively quick and in parallel with others, it probably doesn't matter
+  eclwatchuitests:
+    needs: build-and-setup
+    if: needs.build-and-setup.outputs.cache-hit != 'true' && github.event_name == 'pull_request'
+    timeout-minutes: 10
+    runs-on: ubuntu-20.04
+    steps:
+    - uses: actions/download-artifact@v2
+      with:
+        name: installsetup-artifact
+
+    - name: Prerequisites
+      run: |
+        sudo apt-get update
+        sudo apt-get install -y git wget net-tools
+        sudo apt-get install -y tzdata unzip xvfb libxi6
+        sudo apt-get install -y default-jdk
+        sudo apt-get -yq install bison flex build-essential binutils-dev libldap2-dev libcppunit-dev libicu-dev libxslt1-dev \
+          zlib1g-dev libboost-regex-dev libarchive-dev libv8-dev default-jdk libapr1-dev libaprutil1-dev libiberty-dev \
+          libhiredis-dev libtbb-dev libxalan-c-dev libnuma-dev nodejs libevent-dev libatlas-base-dev libblas-dev python3-dev \
+          default-libmysqlclient-dev libsqlite3-dev libmemcached-dev libcurl4-openssl-dev pkg-config libtool autotools-dev automake \
+          libssl-dev gdb
+        wget https://dl.google.com/linux/direct/google-chrome-stable_current_amd64.deb
+        sudo apt-get install -y ./google-chrome-stable_current_amd64.deb
+        wget https://chromedriver.storage.googleapis.com/2.41/chromedriver_linux64.zip
+        unzip chromedriver_linux64.zip
+        sudo mv chromedriver /usr/bin/chromedriver
+        sudo chown root:root /usr/bin/chromedriver
+        sudo chmod +x /usr/bin/chromedriver
+        wget https://selenium-release.storage.googleapis.com/3.141/selenium-server-standalone-3.141.59.jar
+        wget http://www.java2s.com/Code/JarDownload/testng/testng-6.8.7.jar.zip
+        unzip testng-6.8.7.jar.zip
+        export CLASSPATH=".:../selenium-server-standalone-3.141.59.jar:../testng-6.8.7.jar"
+
+    - name: run
+      run: |
+        export LANG="en_US.UTF-8"
+        sudo update-locale
+        tar --zstd -xf install.tgz
+        source install/opt/HPCCSystems/sbin/hpcc_setenv
+        install/opt/HPCCSystems/etc/init.d/hpcc-init start
+        mkdir tests
+        export CLASSPATH=".:../selenium-server-standalone-3.141.59.jar:../testng-6.8.7.jar"
```

Comment:
those relative paths are a little confusing, I can see end up being right because you cd into tests, but..
would be clearer if used absolute, e.g. used $(realpath selenium-server-standalone-3.141.59.jar)

Reply:
Changed